cmd/cli/account_utxo_records.go

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019. Baidu Inc. All Rights Reserved.
+ */
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/xuperchain/xuperunion/pb"
+)
+
+// QueryUtxoRecordsCommand necessary parmeters for query utxo records
+type QueryUtxoRecordsCommand struct {
+	cli         *Cli
+	cmd         *cobra.Command
+	addr        string
+	utxoItemNum int64
+}
+
+// NewQueryUtxoRecordsCommand an entry to query utxo records
+func NewQueryUtxoRecordsCommand(cli *Cli) *cobra.Command {
+	c := new(QueryUtxoRecordsCommand)
+	c.cli = cli
+	c.cmd = &cobra.Command{
+		Use:   "query-utxo-records",
+		Short: "Get utxo records info of an user.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.TODO()
+			return c.queryUtxoRecords(ctx)
+		},
+	}
+	c.addFlags()
+	return c.cmd
+}
+
+func (c *QueryUtxoRecordsCommand) addFlags() {
+	c.cmd.Flags().StringVar(&c.addr, "address", "", "address")
+	c.cmd.Flags().Int64Var(&c.utxoItemNum, "N", 0, "utxo items to be displayed")
```

Comment:
command flags like "N" should be short for a meaningful word in xchain-cli, like in split utxo, N is short for "num".